### ISSUE TITLE
Collect assister info for each app

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/clean#519]
Let's know who our users are so we can get in touch and make sure they have a good experience. Possible future use case is to send FU reminders or something.

Next step is to mockup the flow/information and think through an easy way to implement.

**Comment by daguar** [comment]
Drop some thoughts here!

**Comment by lippytak** [comment]
v1:
- Simple view before basic info that shows
![screen shot 2015-04-29 at 12 01 09 pm](https://cloud.githubusercontent.com/assets/2533112/7399025/c335e706-ee67-11e4-9d0f-d0427a091e6b.png)

- If user selects 'not on this list' then we reveal a few more questions:
![screen shot 2015-04-29 at 12 02 52 pm](https://cloud.githubusercontent.com/assets/2533112/7399040/daa69980-ee67-11e4-9201-d6219a7c32af.png)

Tasks update in the original comment.

**Comment by daguar** [comment]
One quick thing re above mockups — I find it confusing for the first "you" to refer to the client, and then the second "you" to refer to the assister.

**Comment by daguar** [comment]
Yeah I'm mixed on it - on one side I think low ROI (more due to effort) but I also just saw a case that made me want to know who submitted (the dupes issue, one of which is a Daly City address.)
